_src/static/sass/components/_buttons.scss

```diff
@@ -0,0 +1,371 @@
+    color: palette(grayscale-cool, base);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(grayscale-cool, base);
+        border-color: palette(grayscale-cool, base);
+        color: palette(grayscale, white);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        border-color: palette(primary, dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        border-color: palette(grayscale, x-light);
+        color: palette(grayscale, mid-light);
+    }
+}
+
+
+// ----------------------------
+// #LINK
+// ----------------------------
+.btn-link {
+    background: $transparent;
+    border-color: $transparent;
+    color: palette(primary, mid-dark);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        color: palette(primary, base);
+        text-decoration: underline;
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        color: palette(primary, dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        display: none; // :)
+        color: palette(grayscale, mid-light);
+        text-decoration: none;
+    }
+}
+
+
+// ----------------------------
+// #ELEVATED
+// ----------------------------
+.btn-elevated {
+    background: palette(primary, mid-dark);
+    box-shadow: inset 0 -3px 0 palette(primary, dark);
+    border-color: palette(primary, dark);
+    color: palette(grayscale, white);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(primary, base);
+        box-shadow: inset 0 -3px 0 palette(primary, mid-dark);
+    }
+
+    &:active,
+    &.is-pressed,
+    &.is-active {
+        background: palette(primary, dark);
+        box-shadow: inset 0 3px 0 palette(primary, x-dark);
+    }
+
+    &:disabled,
+    &.is-disabled {
+        background: palette(grayscale, x-light);
+        border-color: palette(grayscale, x-light);
+        box-shadow: inset 0 -1px 0 palette(grayscale, mid-light);
+        color: palette(grayscale, base);
+    }
+}
+
+
+// ----------------------------
+// #COMBO
+// ----------------------------
+.btn-combo {
+    border-radius: 0;
+
+    &:first-of-type {
+        border-top-left-radius: 3px;
+        border-bottom-left-radius: 3px;
+    }
+
+    &:last-of-type {
+        border-top-right-radius: 3px;
+        border-bottom-right-radius: 3px;
+    }
+}
+
+
+// ----------------------------
+// #ICONS
+// ----------------------------
+.has-icon {
+
+}
+
+.has-icon-reverse {
+
+}
+
+
+// ----------------------------
+// #OVERLAY
+// ----------------------------
+.btn-overlay {
+    background: $transparent;
+    border-color: palette(grayscale, white);
+    color: palette(grayscale, white);
+    font-weight: font-weight(semi-bold);
+
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        background: palette(grayscale, white);
+        color: palette(primary, mid-dark);
+    }
+
+    &:active,
```

Comment:
NIT: add stateful Sass comments - https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#sass-comments-for-development